tests/apidocs/util.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import
+
+import os
+from django.test.client import RequestFactory
+from openapi_core import create_spec
+from openapi_core.contrib.django import DjangoOpenAPIRequest, DjangoOpenAPIResponse
+from openapi_core.validation.response.validators import ResponseValidator
+
+from sentry.utils import json
+from sentry.testutils import APITestCase
+
+
+class APIDocsTestCase(APITestCase):
+    def create_schema(self):
+        # get the generated json file
+        path = os.path.join(os.path.dirname(__file__)) + "/openapi-derefed.json"
+        with open(path, "r") as json_file:
+            data = json.load(json_file)
+            # remove DSN and update server to testserver
+            del data["components"]
+
+            return create_spec(data)
+
+    def create_test_response(self, path):
```

Comment:
it's a bit confusing to me that this is called `create_test_response` when it is returning the request. Can we just update the name?